vars/deployApp.groovy

```diff
@@ -21,6 +21,7 @@ def call(Map parameters = [:]) {
     image = parameters.image
     app = parameters.app
     env = parameters.env
+    region = parameters.region ?: 'us-west-1'
```

Comment:
The region parameter is either set to the value of `parameters.region` or in a scenario where `parameters.region` is NULL it is set to a default value of `us-west-1` through the use of the `Elvis operator`.

https://docs.groovy-lang.org/next/html/documentation/#_elvis_operator